src/fenwick.jl

```diff
@@ -0,0 +1,37 @@
+mutable struct FenwickTree{T}
+    BIT::Vector{T}
+    n::Integer
+end
+
+FenwickTree{T}() where T = FenwickTree{T}(0)
+FenwickTree{T}(n::Integer) where T = FenwickTree{T}(fill(zero(T), n), n)
+
+bit(F::FenwickTree{T}) where T = F.BIT
+size(F::FenwickTree{T}) where T = F.n
+
+function update(F::FenwickTree{T}, ind::Int, val::T) where T
```

Comment:
This should be called `update!` as it is mutating

Reply:
Ok!